server/services/store/sqlstore/migrations/migrations_files/000015_add_teams_and_boards.down.sql

```diff
@@ -0,0 +1 @@
+SELECT 1;
```

Comment:
Since the migration UP deletes boards from the blocks and blocks_history table, this empty DOWN means we cannot downgrade and expect a valid database.  Is this intended?  Are we moving away from supporting backward migrations?

Reply:
Both tables can be rebuilt from the boards and boards_history tables, although is not a minor effort. I've left a `SELECT 1` here in the interest of merging fast (on the feature branch) as discussed, but we can complete this before merging if we'd like to support down migrations.

Having said that, we don't really have a mechanism currently to undo a migration AFAIK, and the data migrations would fall outside of this (and the feature will have one as well for the templates) so I don't know how much it makes sense to go through the effort, as we will not be able to migrate down without implementing a system for it besides the feature itself.

Reply:
Ok, makes sense.  Not sure why we're bothering to maintain all those DOWN migrations.

Reply:
Chances are that as they were mandatory for `golang-migrate` we just embraced them (and the data migrations are fairly new). In any case, I've [scheduled an item](https://community-daily.mattermost.com/boards/workspace/b1ni7nprjpfwpb566k1gk8z9nw/886d2280-e88e-4065-8c85-1b62578f192e/f30dcb19-a0e8-45d2-8768-a9489f6e7fe8/crmg8ezy167yu3p1cjrkagctcao) to discuss during tomorrow's dev meeting to make a decision on this